lib/amazon-pricing/rds-price-list.rb

```diff
@@ -152,21 +163,22 @@ def fetch_reserved_rds_instance_pricing2(url, dbs, is_multi_az, is_byol)
     end
 
     def get_rds_reserved_instance_pricing
-       @@DB_TYPE.each do |db|
+      @@DB_TYPE.each do |db|
+        next if @@DB_RI_PRICE_ONLY_FROM_API.include? db
```

Comment:
do you really need this line? also, even if you need to do this, this class variable should be renamed